ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -127,9 +127,9 @@ public NetworkMetrics(MetricRegistry registry) {
 
     transmissionSendPendingTime =
         registry.histogram(MetricRegistry.name(Selector.class, "TransmissionSendPendingTime"));
-    transmissionSendAllTime = registry.histogram(MetricRegistry.name(Selector.class, "TransmissionSendTime"));
+    transmissionSendAllTime = registry.histogram(MetricRegistry.name(Selector.class, "TransmissionSendAllTime"));
```

Comment:
With this change metrics transmissionsendtime and transmissionsendalltime both seem to have same name now. transmissionsendtime one is set on every write and transmissionsendalltime is set on completion of send. Why do you want to make this change?

Reply:
Never mind.. I see the issue now. I interpreted it backwards.

Reply:
TransmissionSendAllTime should be used. And the change is to use `TransmissionSendAllTime `.